arch/Mips/MipsMapping.c

```diff
@@ -902,6 +902,9 @@ static name_map group_name_maps[] = {
 	// generic groups
 	{ MIPS_GRP_INVALID, NULL },
 	{ MIPS_GRP_JUMP, "jump" },
+	{ MIPS_GRP_CALL, "call" },
+	{ MIPS_GRP_RET, "ret" },
+	{ MIPS_GRP_INT, "int" },
 
```

Comment:
Those aren't used currently, so it doesn't much matter.
On Thu, May 12, 2016 at 6:53 AM Nguyen Anh Quynh <notifications@github.com>
wrote:

> In arch/Mips/MipsMapping.c
> <https://github.com/aquynh/capstone/pull/669#discussion_r63023545>:
>
> > @@ -902,6 +902,9 @@ static name_map group_name_maps[] = {
> >  	// generic groups
> >  	{ MIPS_GRP_INVALID, NULL },
> >  	{ MIPS_GRP_JUMP, "jump" },
> > +	{ MIPS_GRP_CALL, "call" },
> > +	{ MIPS_GRP_RET, "ret" },
> > +	{ MIPS_GRP_INT, "int" },
> >
>
> Zach, what is your answer?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/aquynh/capstone/pull/669/files/4c6d791ac8826c5c3dc1d842a2775b29e5721b78#r63023545>
>